library/src/morpheus-master/sass/themes/_light.scss

```diff
@@ -528,7 +528,7 @@
     --sakai-popover-title-background: var(--sakai-background-color-1);
     --sakai-hightlight-color: var(--errorBanner-color);
 
-    --progress-bar-bg: var(--infoBanner-bordercolor);
```

Comment:
In this case, `--sakai-color-blue--lighter-2` doesn't meet the minimum contrast with light text.

![PBsakai_color_blue_lighter_2](https://user-images.githubusercontent.com/81161239/148520877-65569803-13d2-45e6-81f1-4ef9affa1ff4.png)